dali/pipeline/operators/sequence/sequence_rearrange.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_SEQUENCE_SEQUENCE_REARRANGE_H_
+#define DALI_PIPELINE_OPERATORS_SEQUENCE_SEQUENCE_REARRANGE_H_
+
+#include <tuple>
+#include <vector>
+
+#include "dali/pipeline/operators/operator.h"
+
+namespace dali {
+
+static Index GetSeqLength(const std::vector<Index>& seq_shape) {
+  return seq_shape[0];
+}
+
+static std::tuple<std::vector<Index>, Index> GetNewShapeAndElementSize(
+    const std::vector<Index>& in_sample_shape, const std::vector<int>& new_order) {
```

Comment:
This whole tests basically boils down to testing this function. Does it really mandate a separate test?
I could write it, but it will just duplicate some subset of the operator test.